packages/components/pie-switch/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# @justeattakeaway/pie-toggle-switch
+# @justeattakeaway/pie-switch
```

Comment:
Unsure what will happen with this file when the next version packages PR is created. It might require some manual tweaking.